### PR TITLE
Microsoft Teams - enhance documentation

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Description
Added to Microsoft Teams doc the following:
 - Setup examples
 - Troubleshooting
 - Direct messaging and running commands

### CODE REVIEW

**Comment by kirbles19** [comment]
@Itay4 Done with the review of new content.

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 1]
```diff
@@ -23,6 +23,60 @@ The web server for the integration runs within a long-running Docker container.
...
+
+On NGINX, configure the following:
+ - SSL certificate under `ssl_certificate` and `ssl_certificate_key`
+ - The Cortex XSOAR server (including the port) under `proxy_pass`, e.g. `http://mydemistoinstance.com:7000`
+ 
+Follow [Configuring Upstream Servers NGINX guide](https://docs.nginx.com/nginx/admin-guide/security-controls/securing-http-traffic-upstream/#configuring-upstream-servers) for more details.
+
+The port (`7000` in this example), to which the reverse proxy should forward the traffic on HTTP, should be the same port you specify in the integration instance configuration, as the webserver the integration spins up, listens on that port.
+
+![image](./doc_files/RP-NGINX.png)
```
Comment (glicht) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:53]: need full path with commit hash for images

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 2]
```diff
@@ -23,6 +23,60 @@ The web server for the integration runs within a long-running Docker container.
...
+
+![image](./doc_files/InstanceConfig7000.png)
```
Comment (glicht) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:55]: need full path with commit hash for images

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 3]
```diff
@@ -23,6 +23,60 @@ The web server for the integration runs within a long-running Docker container.
...
+
+### 3. Using Apache reverse proxy and Cortex XSOAR engine
+In this configuration, the inbound connection, from Microsoft Teams to Cortex XSOAR, goes through a reverse proxy (e.g. Apache) and possibly a load balancer, which relays the HTTPS requests posted from Microsoft Teams
+to a Cortex XSOAR engine, which can be put in a DMZ, on HTTP.
+
+The port (`7000` in this example), to which the reverse proxy should forward the traffic on HTTP, should be the same port you specify in the integration instance configuration, as the webserver the integration spins up, listens on that port.
```
Comment (glicht) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:61]: Let's also add a link to: https://httpd.apache.org/docs/2.4/howto/reverse_proxy.html

**Comment by Itay4** [comment]
build fails on circleci connection errors, not related to the changes
merging

**Pull request merged**